docs/workflow-controller-configmap.yaml

```diff
@@ -133,6 +133,8 @@ data:
       # save completed workloads to the archived, even if disabled, you'll be able to
       # read from the archive
       archive: false
+      # The name of the cluster I'm running in. This must be unique.
+      clusterName: default-cluster
```

Comment:
This allows you to use one database from multiple clusters.

Reply:
Can this be optional?

Reply:
This is optional and defaults to `default-cluster`